### PR TITLE
 app-crypt/tpm-emulator: Port to EAPI 7 and GLEP 81

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: app-crypt/tpm-emulator/files/tpm-emulator-0.7.5-static-libs.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+https://bugs.gentoo.org/726492
+--- a/tddl/CMakeLists.txt
++++ b/tddl/CMakeLists.txt
+@@ -16,7 +16,6 @@ elseif(WIN32)
+ endif()
+ 
+ install(TARGETS tddl-tpm-emulator DESTINATION ${CMAKE_INSTALL_LIBDIR})
+-install(TARGETS tddl-tpm-emulator_static DESTINATION ${CMAKE_INSTALL_LIBDIR})
```
Comment (SoapGentoo) [inline app-crypt/tpm-emulator/files/tpm-emulator-0.7.5-static-libs.patch:8]: can you completely remove the `tddl-tpm-emulator_static` target too?

File: app-crypt/tpm-emulator/tpm-emulator-0.7.5-r3.ebuild [hunk 2]
```diff
@@ -39,18 +43,18 @@ src_configure() {
 		-DMTM_EMULATOR=$(usex mtm-emulator ON OFF)
 		-DBUILD_DEV=OFF
 	)
-	cmake-utils_src_configure
+	cmake_src_configure
 
 	use modules && ln -s "${BUILD_DIR}/config.h" tpmd_dev/linux
```
Comment (SoapGentoo) [inline app-crypt/tpm-emulator/tpm-emulator-0.7.5-r3.ebuild:48]: `|| die` (which you need to put in a proper `if..fi` block)

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: app-crypt/tpm-emulator/files/tpm-emulator-0.7.5-static-libs.patch (hunk 1 above)
Comment (jsmolic) [reply app-crypt/tpm-emulator/files/tpm-emulator-0.7.5-static-libs.patch:19]: Sure

File: app-crypt/tpm-emulator/tpm-emulator-0.7.5-r3.ebuild (hunk 2 above)
Comment (jsmolic) [reply app-crypt/tpm-emulator/tpm-emulator-0.7.5-r3.ebuild]: Done

**Pull request closed without merging**